src/cstar-cli/cstar/cli/bpd/citizen.py

```diff
@@ -29,6 +29,61 @@ def check_ranking(self):
     citizens_df = citizens_df.apply(next_ranking, axis=1)
 
     print(citizens_df[(citizens_df['ranking_n'] > 100000) & (citizens_df['best_new_ranking_n']  <= 100000)].to_csv())
+  
+  def check_award_received(self):
+    transactions_df = pd.read_csv(self.args.file, sep=';', dtype={"operation_type_c": str, "acquirer_id_s" : str})
+    transactions_df["award_received"] = ""
+    transactions_df["award_amount"] = ""
+    transactions_df["fiscal_code"] = ""
+
+    read_fiscal_code = lambda x : self._read_fiscal_code_by_hashpan(x)
```

Comment:
It is irrelevant from a correctness standpoint but since it's a construct adopted in many places I think we can improve the readability of the codebase by avoiding many unnecessary lambda definitions this way (example):

`transactions_df = transactions_df.apply(self._read_fiscal_code_by_hashpan, axis=1)`

What's your opinion?

Reply:
I totally agree, In this case we have always one argument, so lambdas can be removed

Reply:
Shall I push a commit with the proposed change?

Reply:
@alberto-re-pagopa yes, thanks.. anyway, before merging, we should do a non regression check